### PR TITLE
Snowplow: Add nameTracker option

### PR DESCRIPTION
**Pull request by colmsnowplow**
Adds name tracker option to tracker config

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors/snowplow_v2.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   },
   "extraUrlParams": {
     "url": "${ampdocUrl}",
+    "tna": "${nameTracker}",
```
Comment (paulboocock) [inline extensions/amp-analytics/0.1/vendors/snowplow_v2.json:39]: I appreciate it's called `name_tracker` in the warehouse but it's a bit of an odd name. Does `trackerName` feel more natural or `namespace`?

**Review by paulboocock — APPROVED** [review]

File: extensions/amp-analytics/0.1/vendors/snowplow_v2.json (hunk 1 above)
Comment (colmsnowplow) [reply extensions/amp-analytics/0.1/vendors/snowplow_v2.json:39]: Namespace I _think_ is used for the scope of a config within the AMP domain, so not sure on that one.

I'm totally ok to change to `trackerName`, I went with `nameTracker` just to keep it consistent with how it's named in the tracker protocol/docs.

Comment (paulboocock) [reply extensions/amp-analytics/0.1/vendors/snowplow_v2.json:39]: Let's leave it as nameTracker then, it doesn't make much difference either way

Comment (colmsnowplow) [reply extensions/amp-analytics/0.1/vendors/snowplow_v2.json:39]: Oh, cool will do. :)

**Comment by AlexBenny** [comment]
https://github.com/ampproject/amphtml/pull/34936

**Pull request closed without merging**